Tools/translate.R

```diff
@@ -1,4 +1,12 @@
 #Create the R-nl language files
 
-tools::update_pkg_po("../JASP-Engine/JASP")
+args <- commandArgs(trailingOnly = TRUE)
+
+if (length(args)==0) {
+  cat("Give the root folder of the package or the root of the  R-code of JASP")
+} else if (length(args) == 1) {
+  rootfolder<- args[1]
+}
+
+tools::update_pkg_po(rootfolder)
```

Comment:
So if `length(args) == 0` then `tools::update_pkg_po(rootfolder)` is called, but what is `rootfolder` here?

And why can't you just call `$$_R_HOME/Rscript -e "tools::update_pkg_po('.')"` or something?

Reply:
Yes I changed that in a stop when not 1 argument is given.

Reply:
Wouldn't `$$_R_HOME/Rscript -e "tools::update_pkg_po('$$SOME_BEAUTIFUL_QMAKE_VAR_HERE')"` also break when you do not enter a directory there? And we always run it on the exact same place right?

I think having an extra R file just for this is overkill.

Reply:
It works fine now and I do not think it is an overkill. We can use it in the future for other R translated things.